app/src/components/containers/initialDataContainer/serviceUnavailableScreen.scss

```
@@ -0,0 +1,98 @@
+}
+
+.logo-link {
+  display: inline-block;
+  color: inherit;
+  text-decoration: none;
+}
+
+.logo {
+  width: 140px;
+  height: 30px;
+  background-image: url(~common/img/logo.svg);
```

Stylelint (17.5.0)

[error] Expected quotes around "url" function argument (function-url-quotes)

(function-url-quotes)